DHIS2/cloner/dhis2_clone

```diff
@@ -130,6 +138,19 @@ def magenta(txt):
     return '\x1b[35m%s\x1b[0m' % txt
 
 
+def execute_scripts(scripts_folder):
+    scripts = []
+    files = os.listdir(scripts_folder)
+    for file in files:
```

Comment:
`file` is a reserved name in python. It's not a bug because you are overwriting the reference to the identifier locally in the funcion, but still... maybe use a different name, like `fname`?

Reply:
good catch. I'll use your suggestion